README.md

```diff
@@ -1 +1,4 @@
-# Microverse-start-project
\ No newline at end of file
+# Microverse-start-project
+# Microverse-start-project/index.html
+# Microverse-start-project/index.css
+# Microverse-start-project/.gitignore
```

Comment:
- [ ] You great job 👌, but I see some issues you need to work on it As stated by the previous reviewer 😃kindly modify your README file, you can use this [template](https://github.com/microverseinc/readme-template)⤵️ to update the README file and remember to fill it with your information.💯